grunt-tasks/concat.js

```diff
@@ -25,15 +25,15 @@ module.exports = function(grunt) {
         "<%= jsPath %>/lib/bugform.js",
         "<%= jsPath %>/templates.js"
```

Comment:
Do we need to update this as well?

https://github.com/webcompat/webcompat.com/pull/2394/files#diff-56763bbaaf4762aa0cf49808e18d196eR26

Reply:
Since `templates.js` is also a generated file, I'd put it into the `dist` folder as well.

Reply:
Ah I didn't see it right. Needs to be updated, sure. This made the tests fail I guess.